src/script/descriptor.h

```diff
@@ -13,6 +13,15 @@
 
 #include <vector>
 
+// Cache for pre-computed descriptor things
+class DescriptorCache {
+private:
+    std::map<KeyOriginInfo, CExtPubKey> m_xpubs;
+
+public:
+    bool CacheExtPubKey(const KeyOriginInfo& origin_info, const CExtPubKey& xpub);
```

Comment:
Will this return false in a follow up change?

Reply:
No. I've made it `void`